.pre-commit-config.yaml

```diff
@@ -59,5 +59,6 @@ repos:
     args:
     - "--py3-plus"
     - "--py36-plus"
+    - "--keep-mock"
```

Comment:
first time running into this but ended up having to add this because of https://github.com/asottile/pyupgrade/issues/314 (just didn't feel like i had the cycles to rewrite tests)